### PR TITLE
Mask properties from logging

### PR DESCRIPTION
**Pull request by navis**
For masking some properties, https://groups.google.com/forum/m/#!topic/druid-user/nAj6PPXMNdU

```
druid.startup.logging.maskProperties=["user", "password", "URI"]
```

### CODE REVIEW

File: docs/content/configuration/index.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ All nodes can log debugging information on startup.
 |Property|Description|Default|
 |--------|-----------|-------|
 |`druid.startup.logging.logProperties`|Log all properties on startup (from common.runtime.properties, runtime.properties, and the JVM command line).|false|
+|`druid.startup.logging.maskProperties`|Masks sensitive properties (passwords, for example) containing theses words.|[]|
```
Comment (nishantmonu51) [inline docs/content/configuration/index.md]: How do you think about making the DB password masked by default ?

Comment (navis) [inline docs/content/configuration/index.md]: ok

**Comment by gianm** [comment]
👍

**Comment by nishantmonu51** [comment]
👍

**Comment by drcrallen** [comment]
@nishantmonu51 don't forget to tag PRs

**Comment by drcrallen** [comment]
Note this will probably still show up in the middle manager launch command logging.

**Pull request merged**